internal/cli/atlas/atlas.go

```diff
@@ -54,6 +57,7 @@ func Builder() *cobra.Command {
 		Use:   Use,
 		Short: "MongoDB Atlas operations.",
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			_, _ = fmt.Fprint(os.Stderr, "Deprecated: This command has been deprecated. Please, see https://dochub.mongodb.org/core/migrate-to-atlas-cli to migrate to Atlas CLI.\n\n")
```

Comment:
This is one of the changes

Reply:
can we keep the message consistent?
```suggestion
			_, _ = fmt.Fprintf(os.Stderr, `Command "%s" is deprecated, see https://dochub.mongodb.org/core/migrate-to-atlas-cli to migrate to Atlas CLI.\n\n`, cmd.Path())
```

Reply:
I had to update the message after talking with @dianchenghu